src/App/analytics.ts

```diff
@@ -20,6 +20,11 @@ export default class Analytics implements AnalyticsInterface {
         updateProxy(enableAnalytics);
       }
     );
+
+    this.resetClientId = this.resetClientId.bind(this);
+    this.trackScreenview = this.trackScreenview.bind(this);
+    this.trackEvent = this.trackEvent.bind(this);
+    this.trackError = this.trackError.bind(this);
```

Comment:
These are required in Typescript to prevent this binding to the wrong/global scope AFAIK.